app/queries/get_activity_log_events.rb

```diff
@@ -0,0 +1,26 @@
+class GetActivityLogEvents
+  attr_reader :application_choices
+
+  def initialize(application_choices:)
+    @application_choices = application_choices
+  end
+
+  def call(since: nil)
```

Comment:
Now that this is sufficiently generic, we should be able to use this for individual application timelines :)

Reply:
This would be lovely but we need to establish the activity log and the timeline should be similar things. Currently they are not, notes probably being the biggest difference between the two. But it could perhaps replace `FindStatusChangeAudits`. Some of its `audit.audited_changes.key?('status')` could be turned into scopes.